src/index.html

```diff
@@ -0,0 +1,210 @@
+                    <img src="../assets/m.png" alt="m">
+                </li>
+            </ul>
+            <div class="button-resume">
+                <button type="button" class="btn">
+                    Get my resume
+                </button>
+            </div>
+        </div>
+        <div class="right">
+            <div class="lang">
+
+            </div>
+            <ul class="language-list">
+                <li class="lang">
+                    <p>
+                        Languages
+                    </p>
+                    <img class="imglang" src="../assets/arrowdown.png" alt="arrow">
+                </li>
+                <li class="lang-skill">
+                    <img src="../assets/js.png" alt="js">
+                    <p>
+                        JavaScript
+                    </p>
+                </li>
+                <li class="lang-skill">
+                    <img src="../assets/css.png" alt="html">
+                    <p>
+                        CSS
+                    </p>
+                </li>
+                <li class="lang-skill">
+                    <img src="../assets/html.png" alt="css">
+                    <p>
+                        HTML
+                    </p>
+                </li>
+                <li class="lang">
+                    <p>
+                        FrameWorks
+                    </p>
+                    <img class="imglang" src="../assets/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+                <li class="lang">
+                    <p>
+                        Skills
+                    </p>
+                    <img class="imglang" src="../assets/arrowside.png" alt="">
+                </li>
+                <li class="line"></li>
+            </ul>
+        </div>
+    </section>
+    <section id="contact" class="contact-me">
+        <div class="bg-image">
+            <h2 class="contact-title">Contact me</h2>
+            <p class="contact-text">
+                If you have an application you are interested in developing, a feature
+                that you need built or a project that needs coding. I’d love to help
+                with it
+            </p>
+            <form action="https://formspree.io/f/xqkgeweb" method="POST" class="contact-form">
+                <input type="text" name="name" id="username" placeholder="Your Name" maxlength="30" required />
+                <input type="email" name="email" id="useremail" placeholder="Your Email" required />
+                <textarea name="message" id="usermessage" cols="30" rows="10" maxlength="500"
+                    placeholder="Write your message here" required></textarea>
+                    <p class="alert"></p>
+                <div class="button-resume btn-form">
+                    <button class="btn" type="submit">Get in touch</button>
+                </div>
+            </form>
+        </div>
+    </section>
```

Comment:
- [ ] For the mobile version of the contact section kindly  i recommend you to fix the gap between the about me section end line and the contact me section just like the Figma design .
- The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/222537931-1f452510-6694-4e6b-9389-15ef2646580a.PNG)

- Your Design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/222538180-aee966a7-1871-43ee-9421-992afdf65dc1.PNG)

Reply:
- [ ] This has not been worked on. Kindly implement the grey background.
- [ ] [OPTIONAL] On both views, we have some horizontal scroll bars indicating that some fixed elements are causing overflow. Fixing this will greatly make your page very nice to interact with. It's a good practice for user experience :+1:

Reply:
> * [ ]  For the mobile version of the contact section kindly  i recommend you to fix the gap between the about me section end line and the contact me section just like the Figma design .
> * The Figma design Look likes this :
>   ![Capture](https://user-images.githubusercontent.com/93598189/222537931-1f452510-6694-4e6b-9389-15ef2646580a.PNG)
> * Your Design Look likes this :
>   ![Capture](https://user-images.githubusercontent.com/93598189/222538180-aee966a7-1871-43ee-9421-992afdf65dc1.PNG)

- Your current design
<img width="285" alt="wrong" src="https://user-images.githubusercontent.com/41163571/222924733-6f896ec3-8ff0-4a88-83b7-7c962ce4c264.png">

- Kindly fix the little grey background and margin.